A/armadillo/build_tarballs.jl

```diff
@@ -34,6 +34,13 @@ if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
 fi
 
 cmake .. "${FLAGS[@]}"
+
+# Armadillo doesn't trust that OpenBLAS has LAPACK symbols in it (apparently
+# some distributions don't include them), so we have to manually enable
+# ARMA_USE_LAPACK in the configuration.
+sed -i 's/\/\* #undef ARMA_USE_LAPACK \*\//#define ARMA_USE_LAPACK/' \
+    tmp/include/armadillo_bits/config.hpp
+
```

Comment:
There is no other way to tell CMake to enable it (like a CMake variable) rather than modifying a file it generates?

Reply:
I wish that there was, but not as far as I can tell.  I dug around in the CMake machinery to try and find if there was a way to override it, but basically the Armadillo CMake configuration manually sets `ARMA_USE_LAPACK` to `false` at the beginning, and then only sets it to `true` if a LAPACK distribution it deems suitable is found.  So, I couldn't get away with `-DARMA_USE_LAPACK=true`. :(

Reply:
Have you tried reading the file `cmake_aux/Modules/ARMA_FindLapack.cmake`?  You may have luck by setting the CMake variable `LAPACK_LIBRARY`

Reply:
Thanks!  I always forget that you can bypass `find_library()` checks by just specifying the output.  This is much nicer.  Fixed in 3bcdc18. :+1: